src/Core/Settings.cpp

```diff
@@ -3976,9 +3976,12 @@ Allows to execute `ALTER TABLE ... UPDATE|DELETE|MATERIALIZE INDEX|MATERIALIZE P
 
 Possible values:
 
-- 0 - Mutations execute asynchronously.
-- 1 - The query waits for all mutations to complete on the current server.
-- 2 - The query waits for all mutations to complete on all replicas (if they exist).
+| Value | Description                                                                    |
+|-------|--------------------------------------------------------------------------------|
+| `0`   | Mutations execute asynchronously.                                              |
+| `1`   | The query waits for all mutations to complete on the current server.           |
+| `2`   | The query waits for all mutations to complete on all replicas (if they exist). |
+| `3`   | The query waits only for active replicas.                                      |
```

Comment:
Please add a note that `mutations_sync = 3` is supported only for `SharedMergeTree`